RetrospectiveExtension.Frontend/components/feedbackColumn.tsx

```diff
@@ -143,51 +145,7 @@ export default class FeedbackColumn extends React.Component<FeedbackColumnProps,
     // TODO: Inform user when not all updates are successful due to race conditions.
   };
 
-  public static createFeedbackItemProps = (
-    columnProps: FeedbackColumnProps, 
-    columnItem: IColumnItem,
-    isInteractable: boolean): IFeedbackItemProps => {
-    return {
-      id: columnItem.feedbackItem.id,
-      title: columnItem.feedbackItem.title,
-      createdBy:  columnItem.feedbackItem.createdBy ? columnItem.feedbackItem.createdBy.displayName : null,
-      createdByProfileImage: columnItem.feedbackItem.createdBy ? columnItem.feedbackItem.createdBy._links.avatar.href : null,
-      lastEditedDate: columnItem.feedbackItem.modifedDate ? columnItem.feedbackItem.modifedDate.toString() : '',
-      upvotes: columnItem.feedbackItem.upvotes,
-      timerSecs: columnItem.feedbackItem.timerSecs,
-      timerState: columnItem.feedbackItem.timerstate,
-      timerId: columnItem.feedbackItem.timerId,
-      workflowPhase: columnProps.workflowPhase,
-      accentColor: columnProps.accentColor,
-      iconClass: columnProps.iconClass,
-      createdDate: columnItem.feedbackItem.createdDate.toString(),
-      team: columnProps.team,
-      columnProps: columnProps,
-      columns: columnProps.columns,
-      columnIds: columnProps.columnIds,
-      columnId: columnProps.columnId,
-      boardId: columnProps.boardId,
-      boardTitle: columnProps.boardTitle,
-      defaultActionItemAreaPath: columnProps.defaultActionItemAreaPath,
-      defaultActionItemIteration: columnProps.defaultActionItemIteration,
-      actionItems: columnItem.actionItems,
-      newlyCreated: columnItem.newlyCreated,
-      showAddedAnimation: columnItem.showAddedAnimation,
-      addFeedbackItems: columnProps.addFeedbackItems,
-      removeFeedbackItemFromColumn: columnProps.removeFeedbackItemFromColumn,
-      refreshFeedbackItems: columnProps.refreshFeedbackItems,
-      moveFeedbackItem: FeedbackColumn.moveFeedbackItem,
-      nonHiddenWorkItemTypes: columnProps.nonHiddenWorkItemTypes,
-      allWorkItemTypes: columnProps.allWorkItemTypes,
-      isInteractable: isInteractable,
-      shouldHaveFocus: columnItem.shouldHaveFocus ? true : false,
-      hideFeedbackItems: columnProps.hideFeedbackItems,
-      userIdRef: columnItem.feedbackItem.userIdRef,
-      onVoteCasted: columnProps.onVoteCasted
-    }
-  }
-
-  private renderFeedbackItems = () => {
+  protected renderFeedbackItems = () => {
```

Comment:
Seems to me that this doesn't need to be changed to `protected` right?

Reply:
yeah, changed it back to private. It was some experiments I was doing with the static helper class that I created, but forgot to revert it back to private.